sgt-back-end/server.js

```diff
@@ -0,0 +1,128 @@
+import express from 'express';
+import pg from 'pg';
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+   SELECT *
+      from "grades"
+  `;
+    const result = await db.query(sql);
+    const grade = result.rows;
+    res.status(200).json(grade);
+  } catch (err) {
+    console.error(err);
+    res.status(500).json({ error: 'An unexpected error occurred.' });
+  }
+});
+
+app.post('/api/grades', async (req, res) => {
+  try {
+    const name = req.body.name;
+    const course = req.body.course;
+    const score = req.body.score;
+
+    if (!name || !course || !score) {
+      res.status(400).json({ error: 'Name, course, and score are required fields.' });
```

Comment:
A `score` equal to `0` should not cause a `400 Bad Request`.

Reply:
I updated it! Do I need to send a screenshot of anything for it?

Reply:
@mariannecuevas No screenshot, but it doesn't appear resolved:

---

![image](https://github.com/mariannecuevas/m2-0323-code-solutions/assets/16143771/a74288b1-dd67-4391-bda7-3b81fe807718)